### PR TITLE
ready solution!

### PR DESCRIPTION
**Pull request by job-artem**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+    [
+        (100, 100, [21, 17])
+    ]
+)
+def test_every_4_last_years_for_dog(
+        cat_age: int, dog_age: int, result: list
+) -> None:
+    assert (
+        get_human_age(cat_age, dog_age) == result
+    ), "the next 4 cat years after first 24 give 1 more human year"
```
Comment (Dimosphen1) [inline app/test_main.py:61]: Do not break up these tests to four different functions, you could include them to `pytest.mark.parametrize` at once since the logic is not changed when you pass integers. Besides that, add test cases to check a function with negative input and input with different values for `cat_age` and `dog_age` according to the checklist. You could create a separate function to test against raised errors in case your input is different from integers (e.g. strings, dicts, lists, sets, etc.).

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (28, 28, [3, 2])
+    ]
+)
+def test_every_4_last_years_for_cat(
+        cat_age: int, dog_age: int, result: list
```
Comment (Dimosphen1) [inline app/test_main.py:43]: It would be better to place each argument on a new line to improve the readability.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!
